### PR TITLE
@bentley/imodeljs-native@4.4.2 [release/4.4.x]

### PR DESCRIPTION
**Pull request by anmolshres98**
imodeljs-native 4.4.2 for itwinjs-core 4.4.0 release

Parallel PR: https://github.com/iTwin/itwinjs-core/pull/6403

### CODE REVIEW

File: docs/changehistory/NextVersion.md [hunk 1]
```diff
@@ -6,13 +6,15 @@ publish: false
...
 Table of contents:
 
-- [Tracing API deprecation](#tracing-api-deprecation)
-- [Batched tileset enhancements](#batched-tileset-enhancements)
-  - [Per-model display settings](#per-model-display-settings)
-  - [Support for excluded models](#support-for-excluded-models)
-  - [Batch table property access](#batch-table-property-access)
-- [Geometry](#geometry)
-  - [Range tree search](#range-tree-search)
+- [NextVersion](#nextversion)
```
Comment (aruniverse) [inline docs/changehistory/NextVersion.md:9]: drop this top level (i know it gets auto added by the markdown formatters)

Comment (anmolshres98) [reply docs/changehistory/NextVersion.md]: removed, should be good now

**Review by pmconne — APPROVED** [review]

**Review by aruniverse — APPROVED** [review]

**Review by grigasp — APPROVED** [review]

**Comment by aruniverse** [comment]
@ben-polinsky can you look at failing doc build

File: docs/changehistory/NextVersion.md (hunk 1 above)
Comment (aruniverse) [reply docs/changehistory/NextVersion.md:9]: this is back

Comment (anmolshres98) [reply docs/changehistory/NextVersion.md]: 😮‍💨 my bad. fixed now

**Pull request merged**